src/util/mod.rs

```diff
@@ -14,10 +15,29 @@ pub(crate) fn not_happening() -> Instant {
     *FUTURE
 }
 
+// The goal here is to make a constant "beginning of time" in both Instant and SystemTime
+// that we can use as relative values for the rest of str0m.
+// This is indeed a bit dodgy, but we want str0m's internal idea of time to be completely
+// driven from the external API using `Instant`. What works against us is that Instant can't
+// represent things like UNIX EPOCH (but SystemTime can).
+const HOURS_1: Duration = Duration::from_secs(60);
```

Comment:
Why is this called `HOURS_1` if the value is 1 minute?

Reply:
Very good point.